source/includes/_datasets.md

```diff
@@ -1,21 +1,34 @@
 # Datasets
 
+Datasets are at the core of the platform. A dataset is composed of:
+
+- metadata like a title, a description and keywords, that make users able to discover it in the portal's catalog
+- configurations for processors, visualisation and security, which define the way data will be processed by the platform and made visible to users
+- and of course, the actual data you want to put on your portal (not available via the management API)
+
```

Comment:
I would write:
"Datasets are at the core of the platform.

A dataset is composed of:
- the actual data put on the portal (not available via the management API)
- metadata, meaning the data describing the data itself, like a title, a description and keywords
- configurations for processors, visualisation and security, to define how data will be processed by the platform and made visible to users"

You should really avoid using "of course" in technical documentation. If the "actual data" point is the most obvious/important, simply placing it at the beginning of the list is enough.
I also added more information about metadata for a better understanding.